packages/jest-cli/src/lib/transform.js

```diff
@@ -62,7 +75,7 @@ const writeCacheFile = (cachePath, fileData) => {
   }
 };
 
-const readCacheFile = (filePath, cachePath) => {
+const readCacheFile = (filePath: string, cachePath: string): ?string => {
```

Comment:
for paths can you use `Path` from the `types/Config` module? It makes the code easier to read :)